manual_flagging/manual_update.py

```diff
@@ -152,153 +152,45 @@
 
 # Subset to only City Tri data
 df = df[df["triad_code"] == "1"]
-# Grab only res data
-df_res = df[df["indicator"] == "res"].reset_index(drop=True)
-
-df_res["nbhd"] = df_res["nbhd"].astype(int)
```

Comment:
**[Question, non-blocking]** I see that we're deleting the res update pipeline to make it easier to run the condo pipeline, which makes me kinda nervous. I don't think it's a huge risk since this script is only a stopgap measure anyway, but if we end up needing to make edits to the condo pipeline in the coming days before architecture changes come in (#89), we're going to have to reintroduce this code and remove the condo code and it's going to be a headache. Is there a clean way of using a conditional branch controlled by some type of input flag to tell the script whether to run res vs. condo? Or is the engineering overhead of an approach like that too great to be feasible in the current timeframe?

Reply:
I don't *think* we will need to make changes. However, if we do, I can revert to the other commit that was dealing with the res data. 

A clean way to fix this deletion problem would be to have the architecture changes already completed. I could theoretically build in a bunch of conditionals, but my thinking was that engineering the stop gap wasn't the best use of time since it will mostly be scrapped anyway with the architecture changes. I'm not sure there is a way to partially do it in that we can keep building off of it for the architecture changes. In my head it was simpler to have one commit hash for condos update and one commit hash for res update, but maybe I am thinking about version control in a less-than-ideal way.